README.md

```diff
@@ -892,7 +892,7 @@ where:
   * `fullscan` (`?boolean`) - if `true` then a critical log entry will be skipped
     on potentially long `select`, see [avoiding full scan](doc/select.md#avoiding-full-scan).
   * `mode` (`?string`, `read` or `write`) - if `write` is specified then `select` is
-    performed on master
+    performed on master, default value is read
```

Comment:
There is also undocumented `mode` default in `get` request, let's update its description as well. `count` `mode` is already documented.

Reply:
Added to get too. Check it